src/main/java/io/jenkins/plugins/models/JiraIssue.java

```diff
@@ -0,0 +1,47 @@
+package io.jenkins.plugins.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class JiraIssue {
+    @JsonProperty("key")
+    public String key;
+
+    @JsonProperty("issueType")
+    public String issueType;
+
+    @JsonProperty("priority")
+    public String priority;
+
+    @JsonProperty("status")
+    public String status;
+
+    @JsonProperty("resolution")
+    public String resolution;
+
+    @JsonProperty("summary")
+    public String summary;
+
+    @JsonProperty("assignee")
+    public String assignee;
+
+    @JsonProperty("reporter")
+    public String reporter;
+
+    @JsonProperty("created")
+    public String created;
+
+    @JsonProperty("updated")
+    public String updated;
+
+    @JsonProperty("url")
+    public String getUrl() {
+      return "https://issues.jenkins-ci.org/browse/" + this.key;
+    }
+
+    public JiraIssue() {
```

Comment:
constructors go above methods in java ordering normally, not sure why you even need a constructor here though it should be implied

https://stackoverflow.com/a/10264498/4951015

Reply:
In my opinion, coding style should be enforced by tools, or not enforced.

Is there a java tool like eslint that will yell if thats not the case.

Reply:
Yes there’s a few checkstyle, formatter-maven-plugin, google-java-format